features/admin.users.v1/utils/user-management-utils.ts

```diff
@@ -80,6 +80,39 @@ export class UserManagementUtils {
         return MultipleInvitesDisplayNames[mode];
     }
 
+    /**
+     * Generates code verifier for PKCE.
+     *
+     * @returns A generated code verifier.
+     */
+    public static generateCodeVerifier(): string {
```

Comment:
shall we move these to wso2is/core module?

Reply:
Resolved with e566b44176dd034f329c9cfb9c4a4d39093c7f56